content/_index.html

```diff
@@ -35,6 +35,7 @@ <h4>As a Policy author</h4>
       <p class="text-left font-weight-light">Kubewarden doesn't force you to learn a new Domain Specific Language
       or a query language.</p>
       <p class="text-left font-weight-light">Write policies in your favorite programming language. Reuse your skills and feel instantly productive.
+      <p class="text-left font-weight-light">Develop policies in <a href="https://rust-lang.org" target="_blank">Rust</a>, <a href="https://go.dev" target="_blank">Go</a>, <a href="https://cel.dev" target="_blank">CEL</a>, <a href="https://www.openpolicyagent.org/docs/latest/policy-language/" target="_blank">Rego</a>, and even <a href="https://docs.kubewarden.io/tutorials/writing-policies/wasi/raw-policies" target="_blank">raw policies</a>, described as JSON.
```

Comment:
I would drop the line about the raw policies. They are still written using Go, Rust, CEL, Rego,... but they target JSON documents, they are not written using JSON.

This information (the raw policies) should be shown somewhere else, the key point would be: "Kubewarden can also be used to validate non-Kubernetes admission requests. Its "raw" policies can be used to validate any kind of JSON document"

Reply:
Thanks @flavio. Made some chnages. What does everybody think now? 

- Are the links OK, or should they point to our internal tutorial docs?
- I added the raw policy stuff to an 'Adaptable' section in 'Why use Kubewarden'. Options for 'Adaptable' might be 'General purpose', 'Wide applicability', etc. 'Adaptable' seemed pithy. Thoughts?